tensorflow/compiler/mlir/xla/transforms/materialize_broadcasts.cc

```diff
@@ -255,6 +255,26 @@ bool CreateDynamicBroadcastsForBinaryOp(SrcOp op, PatternRewriter *rewriter,
   return true;
 }
 
+template <typename SrcOp>
+bool GenerateBroadcastForBinaryOp(SrcOp op, PatternRewriter *rewriter,
```

Comment:
The naming is somewhat inconsistent now. How about `CreateStatic...`, `CreateDynamic` and just `CreateBroadcast` for the three cases?

Reply:
ok!fixed!